pgcli/pgcompleter.py

```diff
@@ -29,15 +29,25 @@
     load_config(config_location() + 'config'))
 
 Match = namedtuple('Match', ['completion', 'priority'])
-_SchemaObject = namedtuple('SchemaObject', ['name', 'schema', 'function'])
-def SchemaObject(name, schema=None, function=False):
-    return _SchemaObject(name, schema, function)
+
+_SchemaObject = namedtuple('SchemaObject', 'name schema meta')
+def SchemaObject(name, schema=None, meta=None):
+    return _SchemaObject(name, schema, meta)
 
 _Candidate = namedtuple(
-    'Candidate', ['completion', 'prio', 'meta', 'synonyms', 'prio2']
+    'Candidate', 'completion prio meta synonyms prio2 display'
```

Comment:
I suppose you mean the style change? I probably just thought it looked cleaner and would keep us from running into PEP8's max line length.